[UX/UI] Issue #147 : Fix large space in list of courses CSS

shljessie opened the pull request:
Fixed the unnecessary space below the main title  'MyCourseIndex'  in the Course Selection screen. 
Also made it responsive.


### Previous Screen

**Previous Screen- Web**
<img width="1437" alt="Screen Shot 2020-09-28 at 5 30 24 PM" src="https://user-images.githubusercontent.com/59305253/94408829-4ea15580-01b0-11eb-9729-43546242eb7a.png"  width="100">

**Previous Screen- Tablet**
<img width="642" alt="Screen Shot 2020-09-28 at 5 33 10 PM" src="https://user-images.githubusercontent.com/59305253/94409118-b061bf80-01b0-11eb-8e16-311192777eb6.png" width="100">

**Previous Screen- I phone **
<img width="335" alt="Screen Shot 2020-09-28 at 5 34 25 PM" src="https://user-images.githubusercontent.com/59305253/94409240-ddae6d80-01b0-11eb-932c-4047d633a725.png">

### Fixed Changes

**Fixed Changes Screen**
<img width="1440" alt="Screen Shot 2020-09-28 at 5 32 37 PM" src="https://user-images.githubusercontent.com/59305253/94409067-9d4eef80-01b0-11eb-9625-cbbebd299415.png"  width="100">


**Previous Screen- Tablet**
<img width="387" alt="Screen Shot 2020-09-28 at 5 36 04 PM" src="https://user-images.githubusercontent.com/59305253/94409414-177f7400-01b1-11eb-9381-81df02d3d6f5.png">

**Previous Screen- I phone **
<img width="367" alt="Screen Shot 2020-09-28 at 5 36 26 PM" src="https://user-images.githubusercontent.com/59305253/94409448-2534f980-01b1-11eb-85bd-a784cee0f69d.png">

bayoumi17m commented on app/utils/toke.py:
```diff
@@ -10,7 +10,7 @@
 
 # EMPLOY CONSTANTS USED THROUGHOUT THE FILE AND CAN BE IMPORTED
-sp = spacy.load('en_core_web_lg') #TODO This is for speed
+sp = spacy.load('en_core_web_sm') #TODO This is for speed
```
Could this be changed back? Does not impact functionality too much but will slightly change search results on specific search items.

oscarso2000 replied:
Youcan use sm for testing in your local place, but please ensure that the large package is used in deployment. As magd says, change it back and push it. :)

bayoumi17m replied:
@oscarso2000 We should eventually evaluate the impact this has on the performance as the smaller would be better if it could be used.

shljessie replied:
okay sure got it!

bayoumi17m reviewed (changes requested):
Delete the `.DS_Store` file and the `.vscode/launch.json` as those are specific to you and should not be included in the repo. Otherwise, everything looks good. I did not see any breaking changes, and it seems to run smoothly 😁

oscarso2000 reviewed (changes requested):
As Magd said, any minor changes you've made for personal testing should be reverted back when pushed. (Or just dont git add . and use git add <filename> for specific files instead). DS store shouldn't be here and there seems to be a new yarn.lock file. 

Overall, the changes look great tho. Just fix up the files and we're good.

TrueshotBarrage commented:
- .DS_Store and you got my approval

shljessie commented:
I deleted 'vscode/launch.json ' and fixed the toke.py file. I just can't seem to locate the DS_Store file in my MyCourseIndex folder. Any ideas on how I can find this?

bayoumi17m reviewed (commented):
Sorry! One last change and I will be ready to approve. Can you remove the yarn.lock in the main directory? There seems to be 2 now and I think it might be confusing.
